crates/re_types_core/src/result.rs

```diff
@@ -171,6 +207,9 @@ pub enum DeserializationError {
         backtrace: _Backtrace,
     },
 
+    #[error("Array is smaller than the expected size of {size}")]
+    ArrayInitUnderrun { size: usize, backtrace: _Backtrace },
```

Comment:
the need for this got removed now again, so this variant and associated `array_init_underrun` utility should go as well

Relatedly, it would still be nice to have those unwraps use a localized opt-out with comment, but we can ofc take that another time.

Reply:
I added a localized opt-out, see`deserializer.rs`. The comment is also there but it's inside a `quote!` block and it seems it's stripped during generation.
Any ideas if there is a way to include that comment in the output? ...couldn't find anything in the docs.

Reply:
ah yes that's always a bit annoying: you have to first call `quote_comment` and then insert the result into the actual `quote!`